schemas/data/datasource.schema.json

```diff
@@ -21,10 +21,11 @@
           "description":
             "The unique ID of this data source. This would be provided by the individual or system that created the Datasource.\n"
         },
-        "xdm:name": {
-          "title": "Name",
+        "xdm:code": {
+          "title": "Code",
           "type": "string",
-          "description": "Human readable name of the data source. "
+          "description":
+            "The code is a shortcut to the full @id and at least one of the code or @id can be used. Sometimes, this code refered to as the data source integration code."
```

Comment:
@kstreeter please take care of the typo and make sure that @cdegroot-adobe's explanation of what a `code` is is reflected in the documentation. 

Also, @cdegroot-adobe's explanation made it sound like `code` would exist in addition to `name`, the PR you merged replaces it.